src/MapWindow/MapWindowTraffic.cpp

```diff
@@ -141,13 +141,35 @@ MapWindow::DrawSkyLinesTraffic(Canvas &canvas) const
   if (skylines_data == nullptr)
     return;
 
+  if (DisplaySkyLinesTraffic::OFF ==  GetMapSettings().skylines_traffic_names_enabled){
+    return;
+  }
   canvas.Select(*traffic_look.font);
 
   ScopeLock protect(skylines_data->mutex);
+
   for (auto &i : skylines_data->traffic) {
     PixelPoint pt;
+
     if (render_projection.GeoToScreenIfVisible(i.second.location, pt)) {
+      StaticString<128> buffer; 
+      // Points for the screen coordinates for the icon, name and average climb
+      PixelPoint sc_name;
+      TextInBoxMode mode;
+      mode.shape = LabelShape::OUTLINED;
+
       traffic_look.teammate_icon.Draw(canvas, pt);
+      if (DisplaySkyLinesTraffic::SYMBOL_NAME ==  GetMapSettings().skylines_traffic_names_enabled){
+        const auto name_i = skylines_data->user_names.find(i.first);
+        std::string name = name_i != skylines_data->user_names.end()? name_i->second: std::string();
```

Comment:
Unnecessary use of `std::string`

Reply:
When this is removed, name will be undefined.

Reply:
I didn't tell you to remove the variable. I told you using `std::string` here is unnecessary.

Reply:
By removing std::string this error occur:
src/MapWindow/MapWindowTraffic.cpp:161:9: error: ‘name’ was not declared in this scope
         name = name_i != skylines_data->user_names.end()? name_i->second: std::string();
         ^~~~
src/MapWindow/MapWindowTraffic.cpp:161:9: note: suggested alternative: ‘nice’
         name = name_i != skylines_data->user_names.end()? name_i->second: std::string();
         ^~~~
         nice

Reply:
Are you sure you know how C++ and programming in general works?

Reply:
@MaxKellermann no reason to be mean. you could suggest an alternative instead.

Reply:
This is not mean. I was just asking because the submitter appears to lack knowledge of basic C++. That question is important for me to know because I don't have time to teach C++. But if people believe I'm mean, it's better for me to leave the discussion. Bye.

Reply:
@MaxKellermann yes I am not a specialist in C++.
With some more investigation, I think I found the problem you stated. 
Thanks for your help and time with your reviews.